tests/test_transforms/test_math/test_limit_transform.py

```diff
@@ -0,0 +1,146 @@
+    preprocess = LimitTransform(in_column="target", lower_bound=lower_bound, upper_bound=upper_bound)
+    result = preprocess.fit_transform(ts=ts_check_limit_transform).to_pandas()
+    np.testing.assert_array_almost_equal(
+        result["segment_1"]["target"], result["segment_1"]["target_transformed_" + n_test]
+    )
+
+
+@pytest.mark.parametrize(
+    "lower_bound,upper_bound,n_test", [(None, None, "1"), (None, 9, "2"), (0, None, "3"), (3, 9, "4")]
+)
+def test_fit_transform_non_target(
+    ts_check_limit_transform: TSDataset, lower_bound: float, upper_bound: float, n_test: str
+):
+    """Check forward scaled logit transform works correctly on nontarget column"""
+    preprocess = LimitTransform(in_column="non_target", lower_bound=lower_bound, upper_bound=upper_bound)
+    result = preprocess.fit_transform(ts=ts_check_limit_transform).to_pandas()
+    np.testing.assert_array_almost_equal(
+        result["segment_1"]["non_target"], result["segment_1"]["non_target_transformed_" + n_test]
+    )
+
+
+@pytest.mark.parametrize("lower_bound,upper_bound", [(None, None), (None, 9), (0, None), (0, 9)])
+def test_inverse_transform(ts_check_limit_transform: TSDataset, lower_bound: float, upper_bound: float):
+    """Check that inverse_transform rolls back transform result"""
+    df_copy = ts_check_limit_transform.to_pandas()
+    preprocess = LimitTransform(in_column="target", lower_bound=lower_bound, upper_bound=upper_bound)
+    preprocess.fit_transform(ts=ts_check_limit_transform)
+    preprocess.inverse_transform(ts=ts_check_limit_transform)
+    result = ts_check_limit_transform.to_pandas()
+    np.testing.assert_array_almost_equal(result["segment_1"]["target"], df_copy["segment_1"]["target"])
+
+
+@pytest.mark.parametrize(
+    "lower_bound,upper_bound,n_test", [(-10, 200, "1"), (-10, None, "2"), (-30, 50, "3"), (None, 50, "4"), (-5, 5, "5")]
+)
+def test_fit_transform_values_out_of_borders(
+    ts_check_pipeline_with_limit_transform: TSDataset, lower_bound: float, upper_bound: float, n_test: str
+):
+    """Check that Exception raises when there are values out of bounds"""
+    transform = LimitTransform(in_column="target", lower_bound=lower_bound, upper_bound=upper_bound)
+    if n_test == "2":
+        left_border = lower_bound
+        right_border = np.inf
+    elif n_test == "4":
+        left_border = np.NINF
+        right_border = upper_bound
+    else:
+        left_border = lower_bound
+        right_border = upper_bound
+    with pytest.raises(ValueError, match=f"Detected values out \[{left_border}, {right_border}\]"):
+        transform.fit_transform(ts_check_pipeline_with_limit_transform)
+
+
+def test_pipeline_with_limit_transform(ts_check_pipeline_with_limit_transform: TSDataset):
+    """
+    Check that forecasted target and quantiles are in given range with LimitTransform
+    Check that forecasted target is not in given range without LimitTransform
```

Comment:
A little bit misleading, we don't actually check it, we just show that without transform it would be

Reply:
??? Why don't we actually check it? First assert

Reply:
The purpos of this test is not to check this fact, it is just complimentary check

Reply:
ok